src/main/java/uk/gov/hmcts/reform/judicialbooking/util/ValidationUtil.java

```diff
@@ -77,4 +79,100 @@ public static boolean validateTTL(String strDate) {
         }
         return false;
     }
+
+    public static boolean sanitiseCorrelationId(String inputString) {
+        if (inputString != null && !inputString.isEmpty() && !Pattern.matches(Constants.UUID_PATTERN, inputString)) {
+            throw new BadRequestException(
+                    String.format(
+                            "The input parameter: \"%s\", does not comply with the required pattern",
+                            inputString
+                    ));
+        }
+        return true;
+    }
+
+    public static BookingEntity validateBookingRequest(BookingEntity booking) throws ParseException {
+        validateAppointmentId(booking.getAppointmentId());
+        validateBeginAndEndDates(booking);
+        return booking;
+    }
+
+    public static boolean validateAppointmentId(String appointmentId) {
+        if (appointmentId.isBlank()) {
+            throw new BadRequestException("The Booking appointmentId is invalid or missing.");
+        } else {
+            return true;
+        }
```

Comment:
We are not using the returned boolean value anywhere so this method should not return anything.